tests/keras/backend/backend_test.py

```diff
@@ -1833,6 +1833,14 @@ def test_variable_support_bool_dtype(self):
             with pytest.raises(TypeError):
                 K.variable('', dtype='unsupported')
 
+    def test_clip_with_tensors(self):
+        # Create tensors
+        x_placeholder = K.placeholder(ndim=0)
```

Comment:
You should provide empty shape like you did in next line. You can find the reason in the cntk backend code:
```python
def placeholder(
    # ...
    if not shape:
        if ndim:
            shape = tuple([None for _ in range(ndim)])
    # shape is None and not iterable.
    cntk_shape = [dynamic_dimension if s is None else s for s in shape]
```